rfc9000.md

```diff
@@ -7646,7 +7646,7 @@ The initial contents of this registry are shown in {{iana-error-table}}.
 | 0x0e  | KEY_UPDATE_ERROR          | Invalid packet protection update | {{error-codes}} |
 | 0x0f  | AEAD_LIMIT_REACHED        | Excessive use of packet protection keys | {{error-codes}} |
 | 0x10  | NO_VIABLE_PATH            | No viable network path exists | {{error-codes}} |
-| 0x0100 - 0x01ff | CRYPTO_ERROR     | TLS alert code                | {{error-codes}} |
+| 0x0100-​0x01ff | CRYPTO_ERROR      | TLS alert code                | {{error-codes}} |
```

Comment:
Note that in 7753edbbe54c2c4f3acbbee7d878d5f14769c08a you added a space to avoid awkward table positioning in the txt output.

Reply:
Yes.  This is the result of an exchange with the RFC editor.  What you don't see here is that I added a space here also.  There is a zero-width space after the hyphen.  Of course, our tools erase this thanks to a bug in xml2rfc, but we've been promised a fix to that, so the table layout issue should eventually go away.